lib/auth-middleware/auth_middleware/__init__.py

```diff
@@ -210,7 +210,8 @@ def verify_userauth(id_token, group_name=None, internal_token_only=False):
             #            file1.close()
             lines = base64.b64decode(os.getenv('ROKWIRE_PUB_KEY'))
             keyset = json.loads(lines)
-            target_client_id = os.getenv('ROKWIRE_API_CLIENT_ID')
+
```

Comment:
Line number 199 needs to be updated to `target_client_ids=None`

Reply:
Sorry, I missed it. Updated

Reply:
No issues. Thanks.